webhooks/virtualmachine/v1alpha1/validation/virtualmachine_validator.go

```diff
@@ -738,3 +741,22 @@ func (v validator) isNetworkRestrictedForReadinessProbe(ctx *context.WebhookRequ
 
 	return configMap.Data[isRestrictedNetworkKey] == "true", nil
 }
+
+func (v validator) validateAnnotation(ctx *context.WebhookRequestContext, vm *vmopv1.VirtualMachine) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if ctx.IsPrivilegedAccount {
+		return allErrs
+	}
+
+	annotationPath := field.NewPath("metadata", "annotations")
+	forbiddenAnnotationKey := vmopv1.InstanceIDAnnotation
```

Comment:
Nie: Move these declarations inside the if statement (as they are only used if the annotation is non-empty).